lesson04/assignment/css/_flex.responsive.scss

```diff
@@ -0,0 +1,5 @@
+@mixin flex {
+  display: flex;
```

Comment:
Excellent use of a mixin, especially when you're using this exact set of CSS in multiple areas.